schedule/staging/textmode_install_bsc1208266@64bit-staging.yaml

```diff
@@ -0,0 +1,29 @@
+---
```

Comment:
we don't need a new yaml schedule, we have already a textmode test suite in staging, please use that one.

Reply:
If we use this yaml file, the staging case will be failed as bug has not fix yet.

Reply:
> If we use this yaml file, the staging case will be failed as bug has not fix yet.

because we don't plan to merge the PR until the bug is fixed, it is just preparation.